pkg/kubelet/kubelet.go

```diff
@@ -1152,6 +1163,10 @@ func (kl *Kubelet) registerWithApiserver() {
 			glog.Errorf("Unable to construct api.Node object for kubelet: %v", err)
 			continue
 		}
+
+		// Cache the node object.
+		kl.lastUpdatedNodeObject.Store(node)
```

Comment:
Assuming the initial seed-state is moved to kubelet.Run, should this only be updated if Create() below is successful?

Just a side note, this starts to feel a little fragile - now anywhere someone happens to update the node, they also need to know to populate this field?